### PR TITLE
Added about page and basic content

### PR DESCRIPTION
**Pull request by ecrax**
Added about page, it needs to be implemented into the top bar though.
Someone needs to add some more text to it, so it looks less empty, or I will do it later on.
Also, I tried to orient myself along with the FAQ page, I hope you like it.

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -30,6 +30,9 @@ dependencies:
   cloud_firestore: ^0.14.0+2
   # flutter_svg: ^0.19.0
 
+dependency_overrides:
+  url_launcher_platform_interface: 1.0.8
+
```
Comment (ecrax) [inline pubspec.yaml]: I needed to add this to the pubspec.yaml file to start the project. ([flutter/flutter#68545](flutter/flutter#68545))

**Comment by himanshusharma89** [comment]
@ecrax Can you attach the screenshot of the web app and also can you specify why we need url_launcher_platform_interface here, are you working with dev channel?

**Comment by ecrax** [comment]
### Desktop:
image

### Mobile:
image

-------------------------------------------------------------

I was encountering a bug with url_launcher (as discussed [here](https://github.com/flutter/flutter/issues/68545)) but apparently it was fixed already.

**Comment by himanshusharma89** [comment]
@ecrax I checked the code and saw that there is no navigation to the about screen, can you modify the dashboard.dart.

**Comment by ecrax** [comment]
> @ecrax I checked the code and saw that there is no navigation to the about screen, can you modify the dashboard.dart.

I had to add support for the about page in the top_bar.dart and bottom_bar.dart file but it should be working now.

**Comment by himanshusharma89** [comment]
Perfect @ecrax!

**Comment by himanshusharma89** [comment]
@all-contributors please add @ecrax for code

**Pull request merged**